scio-test/src/main/scala/com/spotify/scio/testing/BigTableMatchers.scala

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2016 Spotify AB.
```

Comment:
Minor: 2018
You can use a `$today.year` variable in IntelliJ's Copyright template.